### ISSUE TITLE
Fix accessibility colors on the front page

### ISSUE THREAD

**Issue by jveillet** [jveillet/jk-demainilpleut#113]
## Goal

Change the colors that are not readable by using more accessible colors using the tool https://dequeuniversity.com/rules/axe/2.2/color-contrast?application=lighthouse.

The colors that need changing, are:
* The grey text for dates on articles
* The grey colors for inactive menu items.

**Comment by jveillet** [comment]
Fixed by #117
